### ISSUE TITLE
Print documents to HTML do not convert md extensions to html

### ISSUE THREAD

**Issue by mimmori** [yzhang-gh/vscode-markdown#1139]
## Problem

Print documents to HTML the wiki style link are not converted to point the html file output.

## How to reproduce

Create the following files in a folder.

parent.md with this content:
test link to [[child.md|child]]

child.md with this content:
bla bla bla

Now start the command "Print documents to HTML" selecting the folder created.

The link in the parent.html file is not working as it points to the end with .md extension.

## Configuration

Default.

## Environment

* Markdown All in One v3.4.3
* VS Code 1.68.1

**Comment by yzhang-gh** [comment]
Unfortunately, this syntax is not supported yet. (This extension currently supports [these syntaxes](https://github.com/yzhang-gh/vscode-markdown#q-which-markdown-syntax-is-supported).)

PRs are welcome.

**Comment by mimmori** [comment]
actually the link with wiki style syntax seems to work.  the link is parsed correctly and the html is generated correctly but the extension of the target file is not updated.  do you think it might be too complicated to take a look at the code and introduce this support too?

**Comment by yzhang-gh** [comment]
I guess you have installed other Markdown extensions that support that syntax? If I type `[[foo.md]]`, it is simply rendered as `[[foo.md]]`.

**Comment by mimmori** [comment]
Yes correct.  in fact I use the Foam plugin which allows me to organize my notes and notes allowing me to link them together using wiki-like links, which are easier to read and manage.

**Comment by mimmori** [comment]
I see that the render of page is managed by markdown-it library. It would seem that to also allow wiki-style link management it is sufficient to add this library as well: https://github.com/kwvanderlinde/markdown-it-wikilinks.

how about giving it a chance?

**Comment by yzhang-gh** [comment]
This extension is compatible with other extensions which use markdown-it if it is done right.

For example, you can check out https://marketplace.visualstudio.com/items?itemName=thomaskoppelaar.markdown-wiki-links-preview, although it may be incompatible with your other extensions.
